cmd/containerboot/main.go

```diff
@@ -312,6 +320,14 @@ authLoop:
 					}
 				}
 			}
+			if cfg.TSEgressIP != "" && (!installedEgressProxy || ipsHaveChanged) && len(n.NetMap.Addresses) > 0 {
+				if err := installEgressForwardingRule(ctx, cfg.TSEgressIP, n.NetMap.Addresses); err != nil {
+					log.Fatalf("installing egress proxy rules: %v", err)
+				}
+				installedEgressProxy = true
```

Comment:
i think the variable isn't really useful anymore
you can make this check be identical to the one above

Reply:
true